src/main/webapp/app/overview/course-exercises/course-exercise-row.component.html

```diff
@@ -37,7 +37,7 @@ <h4 class="fw-medium">{{ exercise.title }}</h4>
                 *ngIf="!isPresentationMode"
             ></jhi-exercise-details-student-actions>
             <div class="exercise-tags col-auto col-sm d-flex justify-content-center mt-2">
-                <h4 class="fw-medium">
+                <h4 class="fw-medium" *ngIf="exercise.releaseDate && dayjs(exercise.releaseDate).isAfter(dayjs())">
```

Comment:
Do we need to convert this here? I think this should be done when receiving the data from the server.

Reply:
I've tested it and it works without it, too.